app/src/main/java/bg/zahov/app/ui/workout/WorkoutViewModelNew.kt

```diff
@@ -1,1006 +0,0 @@
-package bg.zahov.app.ui.workout
```

Comment:
Why is there a `WorkoutViewModelNew` and why is the `new` one being deleted?

Reply:
If the big diff is caused by the renaming it can be in a separate PR.

Reply:
I will place it in a different pr. The point was to just change which vm was used however I didn't know that if I renamed the class within the file when I renamed the file git would consider the whole thing as different